### PR TITLE
Feature/multigroup part5

### PR DESCRIPTION
**Pull request by whaeck**
This adds HeatingNumbers and Kerma records.

### CODE REVIEW

File: python/test/multigroup/Test_NDItk_multigroup_HeatingNumbers.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+            self.assertEqual( self.chunk_string, chunk.to_string() )
+
+            # verify the record
+            self.assertEqual( 'heating', chunk.keyword )
+            self.assertEqual( False, chunk.empty )
+            self.assertEqual( 7, chunk.size )
+
+            values = chunk.values
+            for index in range( chunk.size ) :
```
Comment (tgsaller) [inline python/test/multigroup/Test_NDItk_multigroup_HeatingNumbers.py:44]: I just noticed something... how is this different from the 7 self.assertAlmostEqual for chunk.values[#] above?

File: src/NDItk/MultigroupTable.hpp [hunk 2]
```diff
@@ -123,6 +131,72 @@ class MultigroupTable {
...
+  /**
+   *  @brief Return the primary kerma record
+   */
+  const multigroup::Kerma& primaryKerma() const {
+
+    return this->primary_kerma_;
+  }
+
+  /**
+   *  @brief Return the hkerma record for an outgoing particle
```
Comment (tgsaller) [inline src/NDItk/MultigroupTable.hpp:176]: hkerma

File: src/NDItk/multigroup/HeatingNumbers/src/verify.hpp [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/**
+ *  @brief Verify the weights
+ *
+ *  The following verification tests are performed:
+ *    - there is at least one value
+ *
+ *  @param[in] values    the haeting numbers to be verified
```
Comment (tgsaller) [inline src/NDItk/multigroup/HeatingNumbers/src/verify.hpp:7]: heating

File: python/test/multigroup/Test_NDItk_multigroup_HeatingNumbers.py (hunk 1 above)
Comment (whaeck) [reply python/test/multigroup/Test_NDItk_multigroup_HeatingNumbers.py:44]: Copy/paste. I used to have accessor properties other than 'values' but I replaced I removed most of those without updating the tests.

File: src/NDItk/MultigroupTable.hpp (hunk 2 above)
Comment (whaeck) [reply src/NDItk/MultigroupTable.hpp:176]: consider it changed

File: src/NDItk/multigroup/HeatingNumbers/src/verify.hpp (hunk 3 above)
Comment (whaeck) [reply src/NDItk/multigroup/HeatingNumbers/src/verify.hpp:7]: whoopsie. it's changed now

**Review by tgsaller — APPROVED** [review]

**Pull request merged**